<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>556</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lableLevel">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>학년:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>날짜:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>초5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>초6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>중1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>중2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>중3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>고1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>고2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>고3</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="printBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>506</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>프린트</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>100</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>12</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>56</y>
     <width>491</width>
     <height>441</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>맞은 갯수</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>점수</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>순위</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="setStudent">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>학생 입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delStudent">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>470</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>학생 지우기</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>311</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>31</y>
      <width>311</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addItemBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>성적표 추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delItemBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>성적표 제거</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>저장된 시험지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFileBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>불러오기...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overWriteBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 표 저장</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelDate_2">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>145</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>정답 갯수:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxScoreInput">
   <property name="geometry">
    <rect>
     <x>864</x>
     <y>170</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>제목:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortByStudentName">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>이름으로 정렬</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortByScore">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>점수로 정렬</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyStudent">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>430</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>학생 수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>510</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>표 초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortByGrade">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>310</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>이름 가리기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="printSheet">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>506</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF 파일 생성</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
